OccupancyGridHeader header

GridDataDimension[] dataDimension
float32[] data
